src/builtin/tables.scrbl

```diff
@@ -217,152 +345,294 @@ check:
 end
-check:
-  num-can-drive-col = table: name, age, can-drive, num-can-drive
-    row: "Bob", 12, false, 0
-    row: "Alice", 17, true, 1
-    row: "Eve", 13, false, 1
-  end
+batting-avg-and-slugging = extend batting 
+  using at-bats, singles, doubles, triples, home-runs:
+  batting-average: (singles + doubles + triples + home-runs) / at-bats,
+  slugging-percentage: (singles + (doubles * 2) + 
+    (triples * 3) + (home-runs * 4)) / at-bats
 end
 }
 
-While the reducers found in the @tt{tables} module should cover most all
-use cases, there may be times when one would like to create a reducer of their
-own. To do so, one must construct an object of the following type:
+@(image "src/builtin/baseball.png")
 
+@margin-note{As in @seclink["s:tables:transform"]{@pyret{transform}},
+you must specify which columns will be used to calculate the
+value in the 
+@pyret{extend} expression using the @pyret{using} keyword.}
 
-@type-spec["Reducer" (list "Acc" "InVal" "OutVal")]
-@red-method["one"
-  #:contract (a-arrow (apply Red-of Red-params) "InVal" (a-tuple "Acc" "OutVal"))
-  #:args '(("self" #f) ("value-from-column" #f))
-  #:return (a-tuple "Acc" "OutVal")]
-@red-method["reduce"
-  #:contract (a-arrow (apply Red-of Red-params) "Acc" "InVal" (a-tuple "Acc" "OutVal"))
-  #:args '(("self" #f) ("accumulator" #f) ("value-from-column" #f))
-  #:return (a-tuple "Acc" "OutVal")]
 
+@subsection{Reducers}
 
-Reducers are essentially descriptions of folds (in the list @pyret{fold}
-sense) over table columns. The way reducers are called by the language
-runtime is as follows: the value(s) from the first row are passed to the
-reducer's @pyret{.one} method, which should return a tuple containing both
-any accumulated information needed for the fold and the value which should
-be placed in the new column in that row. The remaining rows are then
-sequentially populated using the reducer's @pyret{.reduce} method, which is
-identical to the @pyret{.one} method except that it receives an additional
-argument which is the previously mentioned accumulated information from the
-previous row.
+A "reducing" column is one whose information is computed from the
+row it is being added to @italic{and one or more of the rows above}
+that row.  This is
+analogous to the @pyret{fold} function for @seclink{lists}.
 
-To illustrate, a @pyret{running-mean} reducer which is equivalent to the
-one provided by the @tt{tables} module could be implemented as follows:
-@pyret-block{
-import tables as TS
-running-mean :: TS.Reducer<{Number; Number}, Number, Number> = {
-  one: lam(n): {{n; 1}; n} end,
-  reduce: lam({sum; count}, n):
-    { {sum + n; count + 1}; (sum + n) / (count + 1) }
-  end
-}
+The simplest examples of reducing use reducers built into Pyret.
+
+In each reducer below, you will need to specify
+a name for the new column and which existing column new value
+will be based on.  
+
+@margin-note{To use @pyret{extend} expressions you will need to
```

Comment:
Not quite -- to use _these particular, predefined reducers_, you need the tables library.  Extend expressions are syntax, and they work regardless of the libraries being used...